src/components/List.css

```diff
@@ -0,0 +1,15 @@
+.soon {
+  background-color: green;
+}
+
+.kind-of-soon {
+  background-color: orange;
+}
+
+.not-too-soon {
+  background-color: blue;
```

Comment:
Color contrast is an important aspect of accessibility on the web. https://www.a11yproject.com/posts/2015-01-05-what-is-color-contrast/

Could we adjust these color values to make the text more readable? Or adjust the text color (i.e. make it white instead of black)?

As it stands these are a little hard to read for me.
Imagine how much harder it would be for vision-impaired folks!

![](https://www.a11yproject.com/posts/2015-01-05-what-is-color-contrast/)

Reply:
This was my main feedback as well. You could use an `rgba` value for the background color...the `a` allows you to add opacity of the color between 0 and 1.0.

Reply:
![image](https://user-images.githubusercontent.com/10378331/107701108-088bf080-6c7e-11eb-870a-06d357a31403.png)
 You can use the dev tools color picker to figure out what that value could be.